lib/cs_guide_web/controllers/brand_controller.ex

```diff
@@ -131,7 +131,7 @@ defmodule CsGuideWeb.BrandController do
     end)
     |> Map.update(:drinks, [], fn drinks ->
       Enum.map(drinks, fn drink ->
-        %{drink | drink_images: Enum.sort_by(drink.drink_images, &(&1.id), &>=/2) }
+        %{drink | drink_images: Enum.sort_by(drink.drink_images, &(&1.id), &<=/2) }
```

Comment:
@SimonLab it could be worth having a comment here to clarify this ...
(_It seems obvious now, but it won't in a few weeks/months time ..._)